libs/content/editor/egui_editor/src/input/canonical.rs

```diff
@@ -121,7 +121,7 @@ pub fn calc(
 ) -> Option<Modification> {
     match event {
         Event::Key { key, pressed: true, modifiers, .. }
-            if matches!(key, Key::ArrowUp | Key::ArrowDown) =>
+            if matches!(key, Key::ArrowUp | Key::ArrowDown) && !cfg!(target_os = "ios") =>
```

Comment:
explain?

Reply:
Since I pass all key events to egui now, I need to be selective about which ones are processed. UITextInteraction already handles keys like arrow up and arrow down, so I don't need to handle it here.

Reply:
What's the behavior without this clause? Is it duplicated?

Reply:
Yes

Reply:
okay sgtm then